cypress-tests/cypress/support/commands.js

```diff
@@ -115,16 +118,18 @@ Cypress.Commands.add("merchantRetrieveCall", (globalState) => {
     logRequestId(response.headers["x-request-id"]);
 
     expect(response.headers["content-type"]).to.include("application/json");
-    expect(response.body.merchant_id).to.equal(merchant_id);
-    expect(response.body.payment_response_hash_key).to.not.be.empty;
-    expect(response.body.publishable_key).to.not.be.empty;
-    expect(response.body.default_profile).to.not.be.empty;
-    expect(response.body.organization_id).to.not.be.empty;
-    globalState.set("organizationId", response.body.organization_id);
-
-    if (globalState.get("publishableKey") === undefined) {
-      globalState.set("publishableKey", response.body.publishable_key);
-    }
+    cy.wrap(response).then(() => {
```

Comment:
move this to `L120` and bring `    expect(response.headers["content-type"]).to.include("application/json");` within `wrap()`